### PR TITLE
47223_userRegister&Login

### PR DESCRIPTION
**Pull request by yasin-buet**
screenshot from 2015-12-01 17 13 17
screenshot from 2015-12-01 17 13 27

### CODE REVIEW

File: resources/views/auth/register.blade.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+					</div>
+					<div class="form-group">
+				        {!! Form::label('email', 'Email', ['class' => 'col-sm-3 control-label']) !!}
+				        {!! Form::email('email') !!}
+					</div>
+					<div class="form-group">
+    					{!! Form::label('password', 'Password', ['class' => 'col-sm-3 control-label']) !!}
+    					{!! Form::password('password') !!}
+					</div>
+						<div class="form-group">
```
Comment (Omid07) [inline resources/views/auth/register.blade.php]: convention

File: resources/views/auth/login.blade.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+				        {!! Form::email('email') !!}
+					</div>
+					<div class="form-group">
+    					{!! Form::label('password','Password', ['class' => 'col-sm-3 control-label']) !!}
+    					{!! Form::password('password') !!}
+					</div>
+					<div class="form-group">
+						<div class="col-sm-offset-3 col-sm-6">
+        					{!! Form::submit('Login') !!}
+        				</div>
```
Comment (Omid07) [inline resources/views/auth/login.blade.php]: convention

File: resources/views/auth/register.blade.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    					{!! Form::label('password_confirmation', 'Confirm Password', ['class' => 'col-sm-3 control-label']) !!}
+    					{!! Form::password('password_confirmation')!!}
+					</div>
+					<div class="form-group">
+						<div class="col-sm-offset-3 col-sm-6">
+        					{!! Form::submit('Register') !!}
+        				</div>
```
Comment (Omid07) [inline resources/views/auth/register.blade.php]: convention

File: resources/views/auth/login.blade.php [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+            </div>
+            <div class="panel-body">
+                @include('common.errors')
+                {!! Form::open(['url' => 'auth/login'], ['class' => 'form-group']) !!}
+                    <div class="form-group">
+                        {!! Form::label('email', 'Email', ['class' => 'col-sm-3 control-label']) !!}
+                        {!! Form::email('email') !!}
+                    </div>
+                    <div class="form-group">
+                        {!! Form::label('password','Password', ['class' => 'col-sm-3 control-label']) !!}
```
Comment (asifbuetcse) [inline resources/views/auth/login.blade.php]: space after 'password'

**Comment by asifbuetcse** [comment]
ok

File: composer.json [hunk 5]
```diff
@@ -7,7 +7,7 @@
     "require": {
         "php": ">=5.5.9",
         "laravel/framework": "5.1.*",
-        "laravelcollective/html": "5.1.*"
+        "laravelcollective/html": "~5.0"
```
Comment (dinhhoanglong91) [inline composer.json]: Use current version `5.1.*`.

File: composer.lock [hunk 6]
```diff
@@ -4,8 +4,10 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#composer-lock-the-lock-file",
         "This file is @generated automatically"
     ],
-    "hash": "9bd5c1c5dfb5bd02facdb1dd88fcf922",
-    "content-hash": "5d22a10ef0e9bc3fa61e71aa29b24879",
+
+    "hash": "9fe98fb23fc07c190c4e294624a06c0d",
```
Comment (dinhhoanglong91) [inline composer.lock]: If composer isn't changed so composer.lock couldn't be changed. Can you revert it to original version.

**Comment by asifbuetcse** [comment]
ok

**Comment by dinhhoanglong91** [comment]
ok

**Pull request merged**